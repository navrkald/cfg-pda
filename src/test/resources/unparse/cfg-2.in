S => bAb
S => AB | ab
A => c
B => d | A
A => e