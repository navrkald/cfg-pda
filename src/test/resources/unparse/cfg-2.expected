S => bAb
S => AB 
S => ab
A => c
B => d
B => A
A => e

